modules/virt-must-gather-usage-targeted-vm-data.adoc

```diff
@@ -13,7 +13,7 @@ When you use the `must-gather` CLI tool to collect {VirtProductName} data, the v
 
 .Environment variables
 
-`NS=<namespace_name>`:: Gather virtual machine details from the namespace that you specify.
+`NS=<namespace_name>`:: Gather virtual machine information, including `virt-launcher` pod details, from the namespace that you specify. Basic VM information, such as the YAML, is still gathered for all namespaces.
```

Comment:
Not sure if the second sentence is necessary because the assembly intro does mention the data that is collected by default. 
If you would like to keep the second sentence, I would suggest rewriting it as:
````
The `VirtualMachine` and `VirtualMachineInstance` CR data is collected for all namespaces". 
````
YAML is the format in which the API objects are defined.